<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VapourSynthProxy</class>
 <widget class="QMainWindow" name="VapourSynthProxy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>93</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VapourSynth Proxy</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineFile"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushFileSel">
        <property name="text">
         <string>Select VS file</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="labelPort">
        <property name="text">
         <string>Port to use: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinboxPort">
        <property name="minimum">
         <number>1024</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>9999</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="labelStatus">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonRun">
        <property name="text">
         <string>Run!</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
